logrotate/test/FilterTest.h

```diff
@@ -10,7 +10,9 @@
 
 #pragma once
 #include <gtest/gtest.h>
+#ifdef __linux__ || __APPLE__  || __unix__
```

Comment:
I think #if something || something_else is valid syntax
but not 
#ifdef something || something_else 
In that case it will have to be 
#if defined(something) || defined(something_else)
I will fix that